django/contrib/admin/static/admin/js/admin/RelatedObjectLookups.js

```diff
@@ -120,7 +120,7 @@
         var selects = django.jQuery(selectsSelector);
         selects.find('option').each(function() {
             if (this.value === objId) {
-                this.innerHTML = newRepr;
+                this.innerText = newRepr;
```

Comment:
I think this may be a separate issue. Could you check if `test_inline_char_fk_edit_with_popup` fails on 1.8? If so, I think we could limit that change to master as a separate commit/ticket. Also, I noticed the regression aspect of that test is non-deterministic based on the position of the `<>` characters which should be fixed.